src/main/java/org/folio/consortia/service/impl/PublicationServiceImpl.java

```diff
@@ -248,8 +273,10 @@ private void updatePublicationsStatus(List<CompletableFuture<PublicationTenantRe
 
     try (var context = new FolioExecutionContextSetter(prepareContextForTenant(centralTenantContext.getTenantId(), folioModuleMetadata, centralTenantContext))) {
```

Comment:
it looks like we're setting context with same context as it was, could be removed